vars/documentationChecker.groovy

```diff
@@ -0,0 +1,51 @@
+
+
+// Add the following code to any Jenkinsfile.
+// It will return with success if there are no changes in the specified docs dir
+// and Will return with success if run on a non Pull Request branch
+
+// documentationChecker("docs")
+
+// docsDir ~~ the subdirectory within your repo that the documentation resides
+def documentationChecker(docsDir) {
+  if (env.CHANGE_ID != null) {
```

Comment:
i'm not sure `CHANGE_ID == null` will ever be the case -- even on master, `CHANGE_ID` is `master`. and since the variable isn't used anywhere, i'm not sure what this `if` block is protecting anyway ;) probably makes sense to remove it.